src/jit/InstList.cpp

```diff
@@ -37,54 +37,27 @@ Index Instruction::resultCount()
     return 1;
 }
 
-BrTableInstruction::~BrTableInstruction()
-{
-    delete[] m_targetLabels;
-}
+BrTableInstruction::~BrTableInstruction() { delete[] m_targetLabels; }
 
-ComplexInstruction::~ComplexInstruction()
-{
-    delete[] params();
-}
+ComplexInstruction::~ComplexInstruction() { delete[] params(); }
```

Comment:
I would keep these two as well, one liner functions should be merged only in classes for getters / setter.